common/xbps-src/libexec/xbps-src-docheck.sh

```diff
@@ -43,6 +43,12 @@ if [ "$make_check" = extended -a "$XBPS_CHECK_PKGS" != full ]; then
     exit 0
 fi
 
+if [ "$make_check" = ci-skip ] && [ "$XBPS_BUILD_ENVIRONMENT" = void-packages-ci ]; then
+    msg_normal \
+        "${pkgname}-${version}_${revision}: skipping check (make_check=ci-skip and XBPS_BUILD_ENVIRONMENT=void-packages-ci) ...\n"
+    exit 0
+fi
+
```

Comment:
I think these should jump at reviewers and submitters; I would use a `msg_warn` and add "make sure to run tests locally" (or something to that effect, my phrasing isn't great) to the message.